winpr/libwinpr/utils/debug.c

```diff
@@ -471,3 +471,29 @@ void winpr_log_backtrace(const char* tag, DWORD level, DWORD size)
 	winpr_backtrace_free(stack);
 }
 
+char* winpr_strerror(DWORD dw, char* dmsg, size_t size) {
```

Comment:
Currently unused, had implemented it while debugging.
But is quite handy so I left it in there, can easily be used as platform independent version of ```strerror```